src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/MediaInfoColumnKey.cs

```diff
@@ -25,82 +27,117 @@ public enum MediaInfoColumnKey
         /// <summary>
         /// Display name.
         /// </summary>
+        [Obsolete("Deprecated since API12; Will be removed in API14.")]
```

Comment:
I would like a dot "." instead of colon ";" better. But it's not an issue.

Reply:
OK. I'll change all the same statement in MediaContent module.